src/elements/content-sidebar/SidebarNavSignButton.tsx

```diff
@@ -21,16 +21,23 @@ export type Props = PlainButtonProps & {
         onComplete: () => void;
         onShow: () => void;
     };
+    CustomFtuxTooltip?: React.ElementType;
```

Comment:
my understanding is that blueprint availability in BUIE is coming soon™ 🤞 

Is this change needed soon or can it wait until after a blueprint->BUIE solution has been implemented? Adding custom components through props to enable blueprint might set an undesirable precedent

Reply:
this is was not designed as a "enable blueprint" change; it does not have any blueprint specifics and will allow any kind of custom component (with the required interface) to be passed and used;

this is a pretty standard & popular pattern in UI libraries that allow users to use their own custom components that may be visually different but also because they may encapsulate custom logic which does not belong inside the library 

this is my use case as well, we do intend to pass a custom component that happens to also use a blueprint component however the most important benefit is that it encapsulates specific logic

Reply:
I understand the pattern but I'm unsure if it's needed in this case. This component and the FTUX tooltip are only used in the All Files page. Rather than adding a custom component, does it make sense to update the regular FTUX tooltip directly in this repo for all use cases? Is there logic being passed that shouldn't exist here?

i.e. if the `CustomFtuxTooltip` change went live in EUA, there wouldn't be any uses of the regular `FtuxTooltip` component. But even if there were additional usages, why wouldn't we want them to be the same for all uses?

separate question: is the custom one using the same `targetingApi` that's currently used? the FTUX experiences are designed so that multiple experiences are not showing at the same time